mediacapture-streams/GUM-required-constraint-with-ideal-value.https.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html>
+<head>
+<title>Ideal value in required constraint in getUserMedia</title>
+<link rel="author" title="Intel" href="http://www.intel.com"/>
+<link rel="help" href="https://w3c.github.io/mediacapture-main/#dfn-fitness-distance">
+</head>
+<body>
+<p class="instructions">When prompted, accept to share your video stream.</p>
+<h1 class="instructions">Description</h1>
+<p class="instructions">This test checks that setting a required constraint
+with an ideal value in getUserMedia works</p>
+
+<div id='log'></div>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script>
+var t = async_test("Tests that setting a required constraint with an ideal value in getUserMedia works");
+t.step(function() {
```

Comment:
Can we change to a promise_test.
Something like:
promise_test(async (t) => {
    const stream = await ...
    ...
}, "Tests that setting a required constraint with an ideal value in getUserMedia works");

async_tests run in parallel of each other and if we add another test later on in this file, this might have undesirable effects.

Reply:
Done.